pkgs/development/python-modules/flit-core/default.nix

```
@@ -20,11 +25,27 @@ buildPythonPackage rec {
     cd flit_core
   '';
 
+  buildPhase = ''
+    runHook preBuild
+    ${python.interpreter} -m flit_core.wheel
```

Don't we need forBuild here?

```suggestion
    ${python.pythonForBuild.interpreter} -m flit_core.wheel
```